### DISCUSSION TITLE
CPU Usage Meter

### DISCUSSION THREAD

**Discussion by evilknish** [SynthstromAudible/DelugeFirmware#1154] [Ideas]
Does anyone know if it’s possible to show a CPU usage meter? I often find I’m working on songs and polyphony will drop out or it will crash on a particularly demanding project. Thanks!

**Comment by n3ptun3s** [comment]
That would be really handy. 
Maybe a percentage value in any corner of the OLED and in the song settings for the seg display that can be enabled in the community settings.

**Comment by sapphire-arches** [comment]
This is challenging because, while measuring the "average" CPU load is possible, the thing that actually determines culling is the maximum load which is very difficult to predict.

**Comment by m-m-adams** [comment]
Included in 1.1 (as much as possible)
